modules/ROOT/pages/bolt/structure-semantics.adoc

```diff
@@ -589,7 +621,11 @@ Since `DateTimeZoneId` does not include the timezone offset, the resolution of t
 The sections below list the changes of structure semantics in versions where they changed.
 Please also check for changes in xref:./message.adoc#messages-summary[Bolt Messages].
 
-[[structure-summary-5]]
+[[structure-summary-60]]
+=== Version 6.0
+** The <<structure-vector, `Vector`>> structure was added.
+
```

Comment:
I think we need to hold to see if we will go with preview release of the protocol.

Reply:
If team networking prefers to hold back the docs until the protocol has been released into the wild server-side (remember, drivers are already out - albeit in alpha phase), then I'm happy to wait with merging this PR until then.